File: Exercises/04-guessing-game.java
int magicNumber = (int) (Math.random() * 10) + 1;
Scanner keyboard = new Scanner(System.in);
System.out.println("Guess the magic number in the range 1-10:");
int userGuess = keyboard.nextInt();
if(userGuess == magicNumber) {
    System.out.println("You got it!");
} else if(userGuess + 1 == magicNumber || userGuess - 1 == magicNumber) {
    System.out.println("So close!");
} else {
    System.out.println("Way off!");
}

File: Exercises/04-guessing-game.js
var magicNumber = parseInt((Math.random() * 10) + 1);
var userGuessString = prompt("Guess the magic number in the range 1-10: ");
var userGuess = parseInt(userGuessString);
if(userGuess == magicNumber) {
    console.log("You got it!");
} else if(userGuess + 1 == magicNumber || userGuess - 1 == magicNumber) {
    console.log("So close!");
} else {
    console.log("Way off!");
}